### PR TITLE
Add dra7xx.md

### PR DESCRIPTION
**Pull request by glneo**
Signed-off-by: Andrew F. Davis <afd@ti.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ Here is a list of manifests for the devices currently supported.
 | MediaTek MT8173 EVB Board | `mt8173-evb.xml` | `mt8173-evb_stable.xml` |
 | ARM Juno board| `juno.xml` | `juno_stable.xml` |
 | Raspberry Pi 3 | `rpi3.xml` | `rpi3_stable.xml` |
+| DRA7xx | `dra7xx.xml` | |
```
Comment (jbech-linaro) [inline README.md]: We should have `dra7xx_stable.xml` too. We've prepared the 2.3.0 release for this device, so everything else is ready. @jforissier, what do you say? Last minute merge this (and manually add the stable) before making the release?

Comment (jforissier) [inline README.md]: Yes we can do that, there's no risk of breaking any other platform so it's fine by me.

**Comment by jforissier** [comment]
@jbech-optee-adm why not add your R-b before merging?

**Comment by jbech-linaro** [comment]
I did on my local branch, but when I pushed I by mistake used the PR as base. I fixed right away, so all should be good now.

**Comment by jforissier** [comment]
Yep all is good, thanks.

**Pull request merged**